types/react-bootstrap-table-next/index.d.ts

```diff
@@ -108,6 +108,8 @@ export interface ColumnDescription<T extends object = any, E = any> {
      * Column header field
      */
     text: string;
+    classes?: string | (cell: T[keyof T], row: T, rowIndex: number, colIndex: number) => string;
+    style?: React.CSSProperties;
```

Comment:
I just checked to API and the style also can accept a function. So how about this:
```suggestion
    style?:
        | React.CSSProperties
        | ((cell: T[keyof T], row: T, rowIndex: number, colIndex: number) => React.CSSProperties);
```